manage_proj_cat_update.php

```diff
@@ -56,6 +56,7 @@
 $f_category_id		= gpc_get_int( 'category_id' );
 $f_name				= trim( gpc_get_string( 'name' ) );
 $f_assigned_to		= gpc_get_int( 'assigned_to', 0 );
+$f_status		= gpc_get_int( 'status', 0 );
```

Comment:
Use constant for default. Rename form / php variable to `cartegory_status` and `$f_category_status`.